subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/AbstractGradleExecuter.java

```diff
@@ -189,8 +186,16 @@ public static void doNotPropagateSystemProperty(String name) {
     protected final GradleDistribution distribution;
     private GradleVersion gradleVersionOverride;
 
-    private boolean debug = Boolean.getBoolean(DEBUG_SYSPROP);
-    private boolean debugLauncher = Boolean.getBoolean(LAUNCHER_DEBUG_SYSPROP);
+    private DefaultJavaDebugOptions debug = createJavaDebugOptions(Boolean.getBoolean(DEBUG_SYSPROP));
```

Comment:
🤔 I wouldn't use the production `DefaultJavaDebugOptions` in our test fixtures. I think it would be simpler to create a new class which doesn't use the Property API to configure our tests.

Reply:
I wasn't sure about that :).
I'll create a new class.